### PR TITLE
Provide unit tests for utils

### PR DESCRIPTION
**Pull request by Kozoriz**
1. Message_meter tests fix (had random crashes)
2. File_system implementation of tests for GetAbsolutePath
3. Enabled 3 test files in CMake (correctly works)

### CODE REVIEW

**Comment by Kozoriz** [comment]
please review: @malirod, @Anatoliy-Leshin , @VVeremjova , @AByzhynar , @OHerasym

**Comment by pestOO** [comment]
Reviewed

File: src/components/utils/test/messagemeter_test.cc [hunk 1]
```diff
@@ -74,7 +74,7 @@ class MessageMeterTest: public ::testing::TestWithParam<TimePair> {
 
     meter.set_time_range(time_range);
     // Get range in msec less than original for correct waitng in tests
```
Comment (VVeremjova) [inline src/components/utils/test/messagemeter_test.cc]: This comment is useless now, it should be deleted or changed

**Comment by OHerasym** [comment]
Reviewed

**Comment by malirod** [comment]
Update the PR subject. Use the "human" style. Don't just leave the branch name.

File: src/components/utils/test/file_system_test.cc [hunk 2]
```diff
@@ -39,10 +40,11 @@ namespace test {
 namespace components {
 namespace utils {
 
+typedef std::vector<std::string> StringArray;
```
Comment (AByzhynar) [inline src/components/utils/test/file_system_test.cc]: Include please "<string> " header

**Comment by AByzhynar** [comment]
reviewed

File: src/components/utils/test/file_system_test.cc (hunk 2 above)
Comment (Kozoriz) [inline src/components/utils/test/file_system_test.cc]: see 32007c0

File: src/components/utils/test/messagemeter_test.cc (hunk 1 above)
Comment (Kozoriz) [inline src/components/utils/test/messagemeter_test.cc]: see a5c447d

**Comment by Anatoliy-Leshin** [comment]
Reviewed

**Comment by VVeremjova** [comment]
Reviewed

**Comment by Kozoriz** [comment]
@malirod  see  	614281f

**Comment by malirod** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
@Kozoriz, please write appropriate pull request name

**Comment by Kozoriz** [comment]
@anosach-luxoft , done

**Comment by anosach-luxoft** [comment]
@Kozoriz, pull request header should not include branch name. Please choose name like **Provide unit tests for utils**. Also, please try to write more extended pull request description.

**Comment by Kozoriz** [comment]
@anosach-luxoft , done

**Comment by VVeremjova** [comment]
reviewed

**Comment by OHerasym** [comment]
reviewed

**Pull request merged**